spring-grpc-test/pom.xml

```diff
@@ -36,16 +35,5 @@
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-testing</artifactId>
 		</dependency>
-
-		<dependency>
```

Comment:
We are not using spring-boot-test and ultimately can not use it in the framework lib as we will have chicken-egg once we moving into Spring Boot.